push-agents/apns-agent/src/main/java/org/aerogear/push/apns/helper/InternalUnifiedPushMessage.java

```diff
@@ -0,0 +1,43 @@
+package org.aerogear.push.apns.helper;
+
+import org.jboss.aerogear.unifiedpush.message.UnifiedPushMessage;
+
+public class InternalUnifiedPushMessage extends UnifiedPushMessage {
+
+    private static final long serialVersionUID = -4220526899054802147L;
+
+    /**
+     * Constant for the "push payload id" key
+     */
+    public static final String PUSH_MESSAGE_ID = "aerogear-push-id";
+
+    private String ipAddress;
```

Comment:
Wouldn't be better to make these final and initialize them in the constructor?

Reply:
is for json deserializtion only